app.rb

```diff
@@ -0,0 +1,142 @@
+require_relative 'person'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+def list_all_books(books)
+  puts 'List of books:'
+  puts
+  books.each_with_index do |book, index|
+    puts "#{index + 1}. Title: #{book.title}, Author: #{book.author}"
+    puts
+  end
+end
+
+def list_all_people(people)
+  puts 'List of people:'
+  puts
+  people.each_with_index do |person, index|
+    puts "#{index + 1}. Name: #{person.name}, ID: #{person.id}"
+    puts
+  end
+end
```

Comment:
- [ OPTIONAL ] For better UX, I suggest that you display a message such as `No book available` when there is no book, people or rental instead of the app displaying as shown in the image

<img width="391" alt="Screenshot 2023-06-09 at 10 57 36" src="https://github.com/MMhaidari/school_library/assets/64774936/aba2f6bc-c171-46e2-b076-66b192980334">